tests/output/test_one_dim_output.py

```diff
@@ -0,0 +1,33 @@
+from firedrake import *
+import numpy as np
+
+def test_plot_constant():
+    mesh = UnitIntervalMesh(1)
+    V = FunctionSpace(mesh, "CG", 1)
+    f = Function(V)
+    f.interpolate(as_ufl(1.0))
+    x_vals, y_vals = one_dimension_plot(f, 10)
+    for y in y_vals:
+        assert np.allclose(1.0, y)
+
+def test_plot_linear():
+    mesh = IntervalMesh(10, 20)
+    V = FunctionSpace(mesh, "CG", 1)
+    f = Function(V)
+    x = SpatialCoordinate(mesh)
+    f.interpolate(x[0] * 2.0)
+    x_vals, y_vals = one_dimension_plot(f, 10)
+    points = np.array([x_vals, y_vals]).T
+    for point in points:
+        assert np.allclose(2.0 * point[0],point[1])
+
+def test_plot_quadratic():
+    mesh = IntervalMesh(10, 20)
+    V = FunctionSpace(mesh, "CG", 2)
+    f = Function(V)
+    x = SpatialCoordinate(mesh)
+    f.interpolate(2 * x[0] - x[0] + 3)
```

Comment:
Do you want an actual quadratic function here?  This is just 2x - x + 3, or x + 3.